optaplanner-core/src/main/java/org/optaplanner/core/config/solver/SolverConfig.java

```diff
@@ -445,6 +456,11 @@ public SolverConfig withSolutionClass(Class<?> solutionClass) {
         return this;
     }
 
+    public SolverConfig withDomainAccessType(DomainAccessType domainAccessType) {
```

Comment:
We like the order of the "with" method (just like the getter/setters) to be in the order of the fields.
so withDomainAccesType() method after withEntityClassList() and also after withEntityClasses().

Motivation: It's just easier to see if a get/set/with method is lacking. Less chaotic.